blathering-barnacles/Bangazon-3
# Dw tick4 orderresource

## Link to Ticket
Closes #4 #10

## Description of Proposed Changes

- Order resource is available to GET/PUT/POST/ and DELETE (when in detail view).
- Products will be displayed when viewing orders (logic is in order_viewset.py). Reference ticket #10
- Customer/Buyer details will be displayed when the query string of "?_include=customers" is added to the end of the URL (i.e. http://localhost:8000/api/v1/orders/?_include=customers)

## Steps to Test

Outline the steps to test
-Go the the order resource, make sure you can GET/POST/PUT/ and DELETE. 
-Make sure you see product details when viewing the order list (refer to ticket #10)
-When viewing the order list, add the string of "?_include=customers" to the URL and make sure you see the customer/buyer information displayed

```sh
git fetch --all
git checkout dw-tick4_orderresource
```


## Impacted Areas in Application
List general components of the application that this PR will affect:

* Order Resource
